CHANGELOG.md

```diff
@@ -1,3 +1,59 @@
+## 2.2.0 / 2023-04-07
+### Features
+- Feat: Adding Fluentd cloudwatch plugin (#586)
+- Feat: Adding an argument for disabling unused controllers (#621) 
+- Feat: Namespace level CRDs and logging with FluentBit Daemonset (#630)
+
+### ENHANCEMENT
+- Update controller-gen to v0.11.3 & update Makefile CRD_OPTIONS (#624)
+- Fluentd add volumes & volumeClaimTemplates (#633)
+- Make v2 importable (#631)
+- Create or patch rbac objects (#635)
+- Remove the deprecated -i flag for go build (#638)
+- Add charts test (#639)
+- Disable stdout output by default in helm (#641)
+- Explain how to disable component controllers in the README (#642)
+- Change gelfShortMessgeKey to gelfShortMessageKey (#643)
+- Enable annotations for service account in fluentbit/fluentd (#647)
+- Update README to mention new namespace FluentBit CRDs (#649)
+- Add fluent-operator.drawio (#650)
+- Updated image of operator architecture (#651)
+- Automatic pushing of fluentbit-debug version images (#656)
+- Add service configurations for the components (#657)
+- Make RBAC comptaible with multiple instances (#658)
+- Add the YAML file for edge nodes to the Helm file (#668)
```

Comment:
`Add support for collecting edge metrics in Helm chart` should be in the features section